src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -818,7 +818,8 @@ DAILY_DEPLOY_EMAIL = '{{ daily_deploy_email }}'
 TRANSIFEX_DETAILS = {
     'organization': 'dimagi',
     'project': {
-        'icds-cas': ['test-project-211', 'icds-cas-aww', 'icds-cas-ls', 'icds-cas-it', 'icds-cas-miscellaneous'],
+        'icds-cas': ['test-project-211', 'icds-cas-aww', 'icds-cas-ls', 'icds-cas-it', 'icds-cas-miscellaneous',
+                     'rtl-test-project'],
```

Comment:
what are your thoughts on making a simple django model for this? then you could just use the django admin UI to update these projects and teams

Reply:
https://github.com/dimagi/commcare-hq/pull/22672/files#diff-1139582615a4af23a9554086b612b78cR115
almost there :)